orion/sites/admin/src/home/home.styl

```diff
@@ -1,6 +1,46 @@
 @import "../static.styl"
 
 #view-home
+  padding: 10px
```

Comment:
please adhere to CSS ordering
width, height, padding/margin, etc.

Do you know the ordering?

Reply:
Umm not sure. So should it be width and height first, then padding and margin and border? And then colors?

Reply:
Yup that's correct!
https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/